### ISSUE TITLE
Change "paint" to "source" and "mapping" to "layer" in OSC messages

### ISSUE THREAD

**Issue by sofian** [mapmapteam/mapmap#451]
OSC messages still use the old nomenclature, which might be confusing. I would change it to the new way of naming things. However I would also keep offering the "paint/mapping" messages for backwards compatibility.

**Comment by aalex** [comment]
Yes: Let's keep backward compatibility - until we hit the 1.0 milestone, at least.
